app/DoctrineMigrations/Version20200730140157.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20200730140157 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('CREATE TABLE organization (id SERIAL NOT NULL, group_id INT DEFAULT NULL, name TEXT NOT NULL, logo TEXT DEFAULT NULL, order_lead_time TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, limit_hour_order TEXT NOT NULL, start_hour_order TEXT NOT NULL, day_of_order_available TEXT NOT NULL, number_of_order_available TEXT NOT NULL, amount_of_subsidy_per_employee_and_order TEXT NOT NULL, coverage_of_delivery_costs_by_the_company_or_the_employee TEXT DEFAULT NULL, delivery_perimeter_expression TEXT NOT NULL, PRIMARY KEY(id))');
+        $this->addSql('CREATE UNIQUE INDEX UNIQ_C1EE637CFE54D947 ON organization (group_id)');
+        $this->addSql('CREATE TABLE customer_group (id SERIAL NOT NULL, code VARCHAR(255) NOT NULL, name VARCHAR(255) NOT NULL, PRIMARY KEY(id))');
```

Comment:
Table `sylius_customer_group` is already there, there is no need to recreate it (see other comment about adding a `table` attribute to Doctrine mapping file)